src/main/kotlin/model/Interventions.kt

```diff
@@ -0,0 +1,144 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.Container
+import com.structurizr.model.Model
+import com.structurizr.model.Person
```

Comment:
This import can be removed as it's unused

Reply:
thank you! fixed in c4b79f9ea098d0823ba5433f479ae52f3d835807